### PR TITLE
Add IMDB search

### PR DESCRIPTION
**Pull request by lucasrcosta**
## Type of change

- [X] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

image

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/search-imdb.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# @raycast.author Lucas Costa
+# @raycast.authorURL https://github.com/lucasrcosta
+# @raycast.description Search IMDB.
+
+# @raycast.icon images/imdb.png
+# @raycast.mode silent
+# @raycast.packageName Web Searches
+# @raycast.schemaVersion 1
+
+# @raycast.argument1 { "type": "text", "placeholder": "Title" }
```
Comment (unnamedd) [inline commands/web-searches/search-imdb.sh:13]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Title", "percentEncoded": true }
```

File: commands/web-searches/search-imdb.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+open "https://www.imdb.com/find?q=${1// /%20}"
```
Comment (unnamedd) [inline commands/web-searches/search-imdb.sh:15]:
```suggestion
open "https://www.imdb.com/find?q=${1}"
```

**Review by unnamedd — COMMENTED** [review]
Hi @lucasrcosta, welcome aboard and thanks for your first contribution.

I made two small comments which will help you to make the conversion from `spaces` to `%20` without need to do it manually.

**Comment by lucasrcosta** [comment]
Thank you @unnamedd ! I had also forgotten the docs...
This thing is awesome! 🎉

**Comment by unnamedd** [comment]
> Thank you @unnamedd ! I had also forgotten the docs...
> This thing is awesome! 🎉

@lucasrcosta for the documentation itself, you don't need to change anything, we have a bot here which will regenerate the commands/README.md based on the metadata at the top of any Script Command!

**Comment by lucasrcosta** [comment]
Oh didn't know that, I'll revert...

**Comment by PitNikola** [comment]
Nice one 🙌

**Pull request merged**